src/Language/Haskell/LSP/Types.hs

```diff
@@ -1,6 +1,8 @@
 module Language.Haskell.LSP.Types
   (
     module Language.Haskell.LSP.TH.DataTypesJSON
+  , module Language.Haskell.LSP.TH.Lens
```

Comment:
I was thinking we should maybe move `Language.Haskell.LSP.Types` into `haskell-lsp-types` and just reexport it from the .cabal file, rather than having this stub module in `haskell-lsp`

Would it be also worthwhile to separate Lens as well, like `Language.Haskell.LSP.Types.Lens`? I've seen a handful of packages on hackage that handle it this way but it would be a very breaking change